modules/indexing/src/test/java/org/apache/ignite/internal/processors/database/IgnitePersistentStoreSchemaLoadTest.java

```diff
@@ -321,27 +361,40 @@ public Person(int id) {
         @QuerySqlField
         protected String name;
 
-        /** {@inheritDoc} */
-        @Override public boolean equals(Object o) {
-            if (this == o)
-                return true;
+        /** */
+        @QuerySqlField
+        protected int city;
+
+        /** */
+        @QuerySqlField
+        protected String str1;
 
-            if (o == null || getClass() != o.getClass())
-                return false;
+        /** */
+        @QuerySqlField(precision = 10, notNull = true)
+        protected String str2;
 
-            IgnitePersistentStoreSchemaLoadTest.Person person = (IgnitePersistentStoreSchemaLoadTest.Person) o;
+        /** */
+        @QuerySqlField
+        protected BigDecimal num1;
 
-            return id == person.id && (name != null ? name.equals(person.name) : person.name == null);
+        /** */
+        @QuerySqlField(precision = 10, scale = 2, notNull = true)
+        protected BigDecimal num2;
 
-        }
+        /** */
+        @QuerySqlField
+        protected Timestamp time1;
 
-        /** {@inheritDoc} */
-        @Override public int hashCode() {
-            int res = id;
+        /** */
+        @QuerySqlField(notNull = true)
+        protected Timestamp time2;
```

Comment:
what about @QuerySqlField(precision = 10, notNull = true) ?

Reply:
After discussion, the test changes were reverted back roll_eyes